operator/templates/index.html

```diff
@@ -11,14 +11,14 @@
         <nav class="navbar navbar-dark bg-primary">
             <a class="navbar-brand">Formula Student Driverless Simulation Operator</a>
             <form class="form-inline">
-                <input class="form-control mr-sm-2" id="access-token" placeholder="Access token" aria-label="Access token">
+                <input class="form-control mr-sm-2" id="access-token" placeholder="Access token" aria-label="Access token" type="password">
```

Comment:
hide password so that we don't have to be scared about showing it during a livestream.